infrastructure/web/lesson_video_buckets.tf

```diff
@@ -0,0 +1,23 @@
+// I want to create a bucket in gcp to hold lesson videos
+resource "google_storage_bucket" "lesson_videos" {
+  name          = "lesson-videos-${var.environment}"
+  location      = var.region
+  force_destroy = var.environment != "production"
+
+  versioning {
+    enabled = false
+  }
+
+  cors {
+    origin          = [var.retool_origin]
+    method          = ["GET", "HEAD", "PUT", "POST", "DELETE"]
+    response_header = ["Content-Type", "x-goog-acl"]
+    max_age_seconds = 3600
+  }
+}
+
+resource "google_storage_bucket_iam_member" "lesson_videos_public" {
+  bucket = google_storage_bucket.lesson_videos.name
+  role   = "roles/storage.objectViewer"
+  member = "allUsers"
```

Comment:
Will there be a `thenational.academy` domain associated with this?

Reply:
I think for now it would be worth either removing this all together or adding a service account. (testing in staging first)